packages/api-gateway-service/src/remote-schema.ts

```diff
@@ -33,7 +33,7 @@ function getSubscriber ( url: string, serviceName: string ): Subscriber {
           'From': 'OP-API-GATEWAY',
         },
         connectionCallback: ( err, res ) => {
-          console.log( err, res );
+          /* Websocket connection established */
```

Comment:
Looks like you are missing configs for err and console for status

Reply:
It was an unused callback with an unnecessary `console.log` which was polluting the server logs.
I've removed altogether it and updated the commit.